rules.py

```diff
@@ -842,37 +842,22 @@ def check(self):
             else:
```

Comment:
You don't have to 'else' here, since if the 'if' condition above is met, the execution flow won't get here.